src/tools/stereoCalib/src/stereoCalibModule.cpp

```diff
@@ -57,11 +47,29 @@ bool stereoCalibModule::close()
 
 bool stereoCalibModule::respond(const Bottle& command, Bottle& reply) 
 {
-    if (command.get(0).asString()=="start") {
-        reply.addString("Starting Calibration...");
-        calibThread->startCalib();
-   }
-    return true;
+    if (command.get(0).asString()=="start")
+	{
+		//create a new folder everytime the it receives the start commmand
+		bool proceed=true;
+        for (int i=1; proceed; i++)
+        {
+            sprintf(dirName,"%s/%s_%.5d",dir.c_str(),"calibImg",i);
```

Comment:
`sprintf` could be unsafe, please use `snprintf` instead.

https://stackoverflow.com/questions/7315936/which-of-sprintf-snprintf-is-more-secure